lib/defaults.js

```diff
@@ -35,12 +37,12 @@ module.exports = {
     function createPackage(name, tag) {
       return {
         ordId: `${regexWithRemoval(name)}:package:${capNamespace}${tag}`,
-        title: `sample title for ${regexWithRemoval(name)}`,
-        shortDescription: "Here is the shortDescription for packages",
-        description: "Here is the description for packages",
+        title: nameWithSpaces(name),
+        shortDescription: "Description for " + nameWithSpaces(name),
+        description: "Description for " + nameWithSpaces(name),
         version: "1.0.0",
-        partOfProducts: [`customer:product:${nameWithDot(name)}:`],
-        vendor: "customer:vendor:SAP:"
+        partOfProducts: [defaultProductOrdId(name)],
+        vendor: "customer:vendor:customer:"
```

Comment:
```suggestion
        vendor: "customer:vendor:Customer:"
```

This is the correct value. It's a reserved Customer vendor, but it should also be documented on the public ORD spec. I couldn't find this on first glance, so I'll add it later.